Add staging directory for kms (#111980)
kubernetes/kubernetes: staging/src/k8s.io/kms/README.md

# KMS

<!-- TODO: Placeholder README. Update with more detail and repo contents once initial implementation is in place. -->

This repository contains the KMS proto APIs.

See https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/3299-kms-v2-improvements for more details.

## Community, discussion, contribution, and support

> The Pod Security Standards are a sub-project of [SIG-Auth](https://github.com/kubernetes/community/tree/master/sig-auth).

Reviewer: ```suggestion
KMS is a sub-project of [SIG-Auth](https://github.com/kubernetes/community/tree/master/sig-auth).
```
